v4/source/system.yaml

```diff
@@ -87,6 +87,35 @@
             // TestEmail
             ok, resp := Client.TestEmail()
 
+  '/file/s3_test':
+    post:
+      tags:
+        - system
+      summary: Test AWS S3 connection
+      description: |
+        Send a test to validate if can connect to AWS S3. Optionally provide a configuration in the request body to test. If no valid configuration is present in the request body the current server configuration will be tested.
+        ##### Permissions
```

Comment:
Can you add a line for `__Minimum server version__: 4.8`?

Reply:
done